### PR TITLE
add release notes style guide link

### LINKED ISSUE #4796: Add a link in the Release note part of the PR template
## Enhancement Task

Add a link in the Release note part of the PR template

### PR DESCRIPTION
**Pull request by TomShawn**
Signed-off-by: TomShawn <41534398+TomShawn@users.noreply.github.com>

### What problem does this PR solve?

Issue Number: Close #4796 

### What is changed and how it works?

```commit-message
add release notes style guide link
```

### Check List

Tests

- No code

### Release note

```release-note
None
```

### CODE REVIEW

**Review by qiancai — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ Related changes
 ### Release note
 
 <!-- A bugfix or a new feature needs a release note. If there is no need release note, just uncomment the below line. -->
+
+Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.
```
Comment (rleungx) [inline .github/pull_request_template.md:70]: keep consistent with TiKV?

Comment (TomShawn) [reply .github/pull_request_template.md]: Done

**Review by rleungx — APPROVED** [review]

**Pull request merged**